### PR TITLE
[docs] Update team options in CMS github issue templates

### PR DESCRIPTION
**Pull request by jilladams**
In .github/ISSUETEMPLATES only: 
* Remove program team from CMS team options
* Combine Plat CMS and Sitewide CMS into new CMS Team label. (Checked with @EWashb & got approval for this change. )
* Remove ⭐ from team labels

Follow up steps required: 
* Update Github [label](https://github.com/department-of-veterans-affairs/va.gov-cms/labels?q=sitewide%20CMS) for `⭐ Sitewide CMS` to become `CMS Team`
* Update [issues](https://github.com/department-of-veterans-affairs/va.gov-cms/labels/Platform%20CMS%20Team) with `Platform CMS Team` label & apply `CMS Team` label 
* Update Github [label](https://github.com/department-of-veterans-affairs/va.gov-cms/labels?q=Public+websites) for `⭐ Public Websites` to remove ⭐ 
* Update Github [label](https://github.com/department-of-veterans-affairs/va.gov-cms/labels?q=Facilities) for `⭐ Facilities`  to remove ⭐ 
 

### Select Team for PR review

- [X] CMS Team
- [ ] `Public websites`
- [ ] `Facilities`
- [ ] `User support`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping the UX writer so they are ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping the UX writer to write and queue content
- [X] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: .ddev/mutagen/mutagen.yml [hunk 1]
```diff
@@ -2,9 +2,8 @@
 # To override this file remove the line above
 # and add your own configuration. If you override it you will
 # probably want to check it in.
-# Please do `ddev mutagen reset` after changing the file.
 # See ddev mutagen docs at
-# https://ddev.readthedocs.io/en/latest/users/install/performance/
```
Comment (jilladams) [inline .ddev/mutagen/mutagen.yml:7]: Not sure how this change made it in. I just ran `ddev start` -- lemme know if I need to revert.

Comment (timcosgrove) [reply .ddev/mutagen/mutagen.yml:7]: Let's revert this one out, yeah. It probably needs to happen but good to keep commits focused on one issue.

**Comment by jilladams** [comment]
I tagged all the eng on CMS for review, but the goal is just awareness and signoff from at least 1 person.

File: .github/ISSUE_TEMPLATE/cms-notification-trigger.md [hunk 2]
```diff
@@ -27,3 +27,11 @@ A trigger is a flag in the CMS that indicates there is an action for a user to c
 **6. Are there any change management implications for this trigger?**
 
 > All requests should be submitted to the Sitewide CMS team for prioritization. Please make sure to notify the Sitewide CMS team after you submit this issue on the #sitewide-cms team slack channel.
+
+### CMS Team
+Please check the team(s) that will do this work.
+
+- [X] `CMS Team`
```
Comment (timcosgrove) [inline .github/ISSUE_TEMPLATE/cms-notification-trigger.md]: Just noting the overlap here. I'm not sure it's an issue, but making our team 'CMS Team' makes it sounds like we own everything :)

File: .ddev/mutagen/mutagen.yml (hunk 1 above)
Comment (jilladams) [reply .ddev/mutagen/mutagen.yml]: ✅

**Review by timcosgrove — APPROVED** [review]
You get a ⭐ for all the removed ⭐

**Review by timcosgrove — APPROVED** [review]
This is exciting!

**Pull request merged**